source/reflectionzeug/source/IniPropertySerializer.cpp

```diff
@@ -75,17 +75,17 @@ void PropertySerializer::serializeGroup(const PropertyGroup & group)
         });

```

Comment:
i'm unsure about the overall design and use of lambdas - can you please provide details for the design decision